src/node/interop/interop_client.js

```diff
@@ -135,7 +135,7 @@ function largeUnary(client, done) {
 function clientStreaming(client, done) {
   var call = client.streamingInputCall(function(err, resp) {
     assert.ifError(err);
-    assert.strictEqual(resp.aggregated_payload_size, 74922);
+    assert.strictEqual(resp.aggregatedPayloadSize, 74922);
```

Comment:
I'm not sure what you mean when you say that `bytes_field` "never worked" when encoding. The fact that all of the previous tests used snake_case field names, and that you had to change test files to get them to work, indicates that snake_case field names did work previously.